### PR TITLE
Templates update

### PR DESCRIPTION
**Pull request by PreslavKozovski**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -59,6 +59,7 @@ tkui: "Telerik Kendo UI"
 tuin: "Telerik UI for NativeScript"
 
 # Telerik UI services
+ab-sab: "Code (AppBuilder)"
```
Comment (TsvetinaIvanova) [inline _config.yml]: Do we need to tie the two? We can just use {{site.ab-s}} ({{site.ab}}) in the text.

File: _config.yml [hunk 2]
```diff
@@ -92,6 +93,14 @@ android-store: "Google Play"
...
 mvs: "Microsoft Visual Studio"
 
+# Aliases for the AB clients
+"ab-web": "in-browser client"
+"ab-win": "classic Windows desktop client"
+"ab-vse": "extension for Visual Studio"
+"ab-cli": "command-line interface"
+"ab-sp": "package for Sublime Text"
+"ab-proton": "universal desktop client"
+
```
Comment (TsvetinaIvanova) [inline _config.yml:103]: Not sure about these, too. First, with this spelling, they cannot be used in the beginning of a sentence or in a title. I suggest that we use the text replacement feature only for names of products, brands, etc.

File: _templates/faqs.md [hunk 3]
```diff
@@ -9,14 +9,26 @@ tags: [Relevant keywords. A list of the available KB tags is available in the ta
...
 
 ## Environment
 <table>
   <tr>
     <td>Service</td>
-    <td>Code (AppBuilder), Views, App Distribution (AppManager), Data (Backend Services), Users, Notifications, Business Logic, Analytics, Feedback, Samples, Telerik Platform General, Third-party product</td>
+    <td>
+{{site.ab-sab}}, <!--Code (AppBuilder)-->
+{{site.sb-s}}, <!--Views-->
+App Distribution ({{site.am}}), <!--AppManager-->
```
Comment (TsvetinaIvanova) [inline _templates/faqs.md]: Add App Distribution to _config.yml and use it here.

File: _templates/faqs.md [hunk 4]
```diff
@@ -9,14 +9,26 @@ tags: [Relevant keywords. A list of the available KB tags is available in the ta
...
-
 ---
 
 ## Environment
 <table>
   <tr>
     <td>Service</td>
-    <td>Code (AppBuilder), Views, App Distribution (AppManager), Data (Backend Services), Users, Notifications, Business Logic, Analytics, Feedback, Samples, Telerik Platform General, Third-party product</td>
+    <td>
+{{site.ab-sab}}, <!--Code (AppBuilder)-->
```
Comment (TsvetinaIvanova) [inline _templates/faqs.md]: If you agree with my comments about the _config.yml, make this in two parts (like the BS one).

File: _templates/faqs.md [hunk 5]
```diff
@@ -28,26 +40,26 @@ pitsid: [A number, representing the public issue tracker ID associated with the
...
   <tr>
-    <td>Code (AppBuilder) client</td>
-    <td>In-Browser, Extension for Visual Studio, Universal Desktop, Windows, Command-line interface, Sublime Text</td>
+    <td>{{site.ab-sab}}<!--Code (AppBuilder)--> client</td>
+    <td>{{site.ab-web}}, {{site.ab-vse}}, {{site.ab-proton}}, {{site.ab-win}}, {{site.ab-cli}}, {{site.ab-sp}}</td>
   </tr>
   <tr>
     <td>Mobile OS</td>
-    <td>iOS, Android, Windows Phone 8</td>
+    <td>iOS, Android, {{site.wp8}}</td>
```
Comment (TsvetinaIvanova) [inline _templates/faqs.md]: Not sure if we need this. Windows Phone is an old product and its name won't change. Additionally, I doubt if we will ever see Windows Universal in AppBuilder :)

File: _config.yml (hunk 1 above)
Comment (PreslavKozovski) [reply _config.yml]: This change is proposed by another colleague, both are ok for me. We can vote what is better.

File: _config.yml (hunk 2 above)
Comment (PreslavKozovski) [reply _config.yml:103]: Got these from Icenium/docs/_config.yml we can capitalize or remove them. We can also vote for this :D.

File: _config.yml (hunk 1 above)
Comment (ggarabedian) [reply _config.yml]: I agree with Tsvetina here. This way you'll have to make a lot more modifications if this is changed at some point.

File: _config.yml (hunk 2 above)
Comment (PreslavKozovski) [reply _config.yml:103]: @Mike4o @AntonDobrev @mkapostolov Could you please share your opinion about this.

Comment (Mike4o) [reply _config.yml:103]: I have always referred to AppBuilder clients in capitalized form, so I would go for capitalizing all of the above. Thus, we can agree to use them capitalized not only in titles but everywhere. What do you think?

**Pull request merged**